src/styles/blocks/icons.scss

```diff
@@ -0,0 +1,17 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+}
+
+.icons__icon--phone {
```

Comment:
Wrong BEM: `icons__icon` indicated an element of the block `icons` but this block doesn't exist
This modifier could simply be named `icon--phone` and all would be fine

Reply:
Done